base_class.py

```diff
@@ -0,0 +1,43 @@
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver import ActionChains
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+
+
+class BaseClass(object):
+    def __init__(self, driver):
+        self.driver = driver
+        self.wait = WebDriverWait(self.driver, 45)
+
+    def wait_for_element(self, selector):

```

Comment:
her fonksiyonun altına o fonksiyonda neler yapılıyorsa ingilizce bir şekilde yazalım (Bir kaçtane fonksiyonunda yazılmış diğer hepsi onlar gibi olmalı)
Tüm fonksiyonlar için bu degisiklikleri yapalım